subsys/bluetooth/mesh/CMakeLists.txt

```diff
@@ -41,7 +41,11 @@ zephyr_library_sources_ifdef(CONFIG_BT_MESH_PB_ADV pb_adv.c)
 
 zephyr_library_sources_ifdef(CONFIG_BT_MESH_PB_GATT pb_gatt.c)
 
-zephyr_library_sources_ifdef(CONFIG_BT_MESH_PROXY proxy.c)
+zephyr_library_sources_ifdef(CONFIG_BT_MESH_GATT_SERVER gatt_services.c)
+
+if (CONFIG_BT_MESH_GATT_SERVER OR CONFIG_BT_MESH_GATT)
+	zephyr_library_sources(proxy_msg.c)
+endif()
 
```

Comment:
Need to make it clear, CONFIG_BT_MESH_GATT is most suitable here. This is because this is selected by pb-gatt-srv and proxy-srv, and the future pb-gatt-cli and proxy-cli are selected. @trond-snekvik Am I right?

Reply:
I agree :+1: